### PR TITLE
Convert Polkadot Launch config to Zombienet

### PR DESCRIPTION
**Pull request by wirednkod**
Convert a given config file from project `previous-launch` to zombienet config `.toml` and `.json`. Files are placed in same dir as the given file with the addition of `-zombie` to the filename.

-Missing (and ofc TODOs): Add support convertion for genesis customization / hrmpChannels and types

### CODE REVIEW

File: src/types.d.ts [hunk 1]
```diff
@@ -255,3 +255,53 @@ export interface Resources {
...
+            validation_upgrade_frequency: number;
+            validation_upgrade_delay: number;
+          };
+        };
+      };
+    };
+  };
+}
+
+interface ParaChainConfig {
```
Comment (pepoviola) [inline src/types.d.ts:285]: We have a similar intergace `ParachainConfig` we should use a prefix like `PL_` or maybe move those to a diff types.d file.

File: src/types.d.ts [hunk 2]
```diff
@@ -255,3 +255,53 @@ export interface Resources {
...
+  bin?: string;
+  id?: string;
+  port?: string;
+  balance?: string;
+  nodes: [NodesConfig];
+}
+
+interface HrmpChannels {
```
Comment (pepoviola) [inline src/types.d.ts:293]: same here.

File: src/cli.ts [hunk 3]
```diff
@@ -119,6 +122,117 @@ const latestPolkadotReleaseURL = async (
...
+async function convertInput(filePath: string) {
+  const { fullPath, fileName, extension } = getFilePathNameExt(filePath);
+
+  const convertedJson = await readInputFile(extension, filePath);
+
+  stream = startStream(`${fullPath}/${fileName}-zombie.toml`);
+
+  const { relaychain, parachains, simpleParachains } = convertedJson;
+
+  if (relaychain) {
```
Comment (pepoviola) [inline src/cli.ts:188]: Hey @wirednkod, we can also simplify the conversion by work with `json` directly in js and the use something like [json2toml](https://www.npmjs.com/package/json2toml) to output `toml`. What do you think?

Comment (wirednkod) [reply src/cli.ts:188]: 🤔 Looks more decent than the solution below. Thank you @pepoviola - will make the convertion

**Review by pepoviola — APPROVED** [review]
Looks good!! thanks!

**Pull request merged**